projects/igniteui-angular/schematics/ng-add/index.spec.ts

```diff
@@ -65,18 +68,31 @@ describe('ng-add schematics', () => {
     expect(pkgJsonData.dependencies['hammerjs']).toBeTruthy();
   });
 
-  it('should add hammer.js to the workspace', () => {
+  it('should add hammer.js to the main.ts file', () => {
     runner.runSchematic('ng-add', { normalizeCss: false }, tree);
+    const mainTs = tree.read('src/main.ts').toString();
+    expect(mainTs).toContain('hammerjs');
```

Comment:
You know exactly what should be added, so you can probably make the check more specific. Should contain something like `import 'hammerjs';`, right?